src/pages/edge/identity-for-edge-network/index.md

```diff
@@ -11,7 +11,9 @@ import Tabs from './tabs/index.md'
 
 # Identity for Edge Network
 
-The Adobe Experience Platform Identity mobile extension enables identity management from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../edge-network/index.md).
+The Adobe Experience Platform Identity mobile extension enables identity management, including the Experience Cloud ID (ECID), from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../edge-network/index.md).
```

Comment:
"The Adobe Experience Platform Identity mobile extension enables identity management" - do we want to be more specific it is regarding to the "The Adobe Experience Platform Identity for Edge Network extension enables identity management, ..."?